addons/mail/static/src/components/file_uploader/file_uploader.js

```diff
@@ -97,14 +97,25 @@ class FileUploader extends Component {
      */
     async _performUpload(files) {
         for (const file of files) {
-            const response = await this.env.browser.fetch('/web/binary/upload_attachment', {
-                method: 'POST',
-                body: this._createFormData(file),
-            });
-            let html = await response.text();
-            const template = document.createElement('template');
-            template.innerHTML = html.trim();
-            window.eval(template.content.firstChild.textContent);
+            const tempAttachment = this.env.models['mail.attachment'].find(attachment =>
+                attachment.filename === file.name);
+            const signal = tempAttachment.uploadController.signal;
+
+            try {
+                const response = await this.env.browser.fetch('/web/binary/upload_attachment', {
+                    method: 'POST',
+                    body: this._createFormData(file),
+                    signal,
```

Comment:
ES6 shorthand way of passing prop is inappropriate here, would be best to not make a variable `signal` and just pass `signal: uploadingAttachment.uploadingAbortController.signal`.

Reply:
why is it inappropriate ?

Reply:
If you make a variable just so you can pass it like this, then you don't need to use shorthand way of passing prop.

Reply:
I see